cli/test/smokehouse/frontends/smokehouse-bin.js

```diff
@@ -70,7 +67,6 @@ function getDefinitionsToRun(allTestDefns, requestedIds, {invertMatch}) {
   });
   if (unmatchedIds.length) {
     console.log(log.redify(`Smoketests not found for: ${unmatchedIds.join(' ')}`));
-    console.log(`Check test exclusions (${requestedIds.join(' ')})\n`);
```

Comment:
seems like logging in this case would still be helpful? Though it might have to pipe `excludedTests` into here

Reply:
Added that log line back.